pyproject.toml

```diff
@@ -29,6 +29,7 @@ pydantic = "^2.6.0"
 psycopg2 = { version = "^2.9.9", optional = true }
 psycopg2-binary = { version = "^2.9.9", optional = true }
 pymysql = { version = "^1.1.0", optional = true }
+python-lorem = "^1.3.0.post1"
```

Comment:
This is a dependency for running test so it should be in `tool.poetry.group.dev.dependencies` section.

Reply:
done